app/imports/components/comments/Comments.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+import { Comment } from '/imports/components/comments/Comment.jsx';
+
+export const Comments = ({comments, commentsLimit, postId}) => {
```

Comment:
do this variable values not pass thru as `props`? If so better to use a consistent pattern